log4j-api-test/src/test/java/org/apache/logging/log4j/message/ParameterFormatterTest.java

```diff
@@ -17,156 +17,167 @@
 package org.apache.logging.log4j.message;
 
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
 
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.MethodSource;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.assertj.core.api.Assertions.assertThat;
 
 /**
  * Tests {@link ParameterFormatter}.
  */
 public class ParameterFormatterTest {
 
```

Comment:
How do I know you didn't change behavior when you have modified the tests?

Reply:
I copied them intact into a `@ParametrizedTest`s. I will create a thread for the ones I either changed or added.